incubator/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -2064,6 +2093,17 @@ private void openSubscribeStreams(
         {
             final Long2ObjectHashMap<List<Subscription>> subscriptionsByRouteId = new Long2ObjectHashMap<>();
 
+            if (!adminSubscribe)
```

Comment:
The idea of `adminSubscribe` is good, but `admin` is only one use case and has no subscription id, and we've discovered we need to handle this for non-admin subscriptions during reconnect.

Suggest renaming to `implicitSubscribe` or similar to cover both use cases; admin and reconnect.